cmd/webhook/main.go

```diff
@@ -321,12 +325,13 @@ func NewConversionController(ctx context.Context, cmw configmap.Watcher) *contro
 					sourcesv1alpha2_: &sourcesv1alpha2.PingSource{},
 				},
 			},
-			sourcesv1alpha2.Kind("SinkBinding"): {
+			sourcesv1beta1.Kind("SinkBinding"): {
 				DefinitionName: sources.SinkBindingResource.String(),
 				HubVersion:     sourcesv1alpha1_,
```

Comment:
Shouldn't this be sourcesv1beta1_?

Reply:
nope, I set v1alpha1 to be the hub version, this will be the one that knows how to convert to all other versions. 
Look at one comment Ville added in the issue, for a clearer explanation. In your case, your hub will be v1alpha2

Reply:
Oh I see. Got it.